contrib/drivers/huawei/dorado/metrics.go

```diff
@@ -0,0 +1,232 @@
+// Copyright (c) 2019 The OpenSDS Authors.
+//
+//    Licensed under the Apache License, Version 2.0 (the "License"); you may
+//    not use this file except in compliance with the License. You may obtain
+//    a copy of the License at
+//
+//         http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+//    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+//    License for the specific language governing permissions and limitations
+//    under the License.
+package dorado
+
+import (
+	"strconv"
+	"time"
+
+	log "github.com/golang/glog"
+	. "github.com/opensds/opensds/contrib/drivers/utils/config"
+	"github.com/opensds/opensds/pkg/model"
+	"github.com/opensds/opensds/pkg/utils/config"
+	"gopkg.in/yaml.v2"
+)
+
+// Todo: Move this Yaml config to a file
+
+var data = `
+resources:
+  - resource: volume
+    metrics:
+      - iops
+      - read_throughput
+      - write_throughput
+      - response_time
+      - service_time
+      - utilization_prcnt
```

Comment:
Can you add comments on the naming of these metrics on the storage system if they are called something else?

Reply:
Add a "TODO" to add resources for "disk" and "port".